<script>
    const promise = new Promise((resolve, reject) => {
        reject('foo')
    })

    promise.catch(console.log)
</script>